controllers/users.js

```diff
@@ -75,6 +78,43 @@ const getUsers = async (req, res) => {
   }
 };
 
+/**
+ * Returns the usernames of inactive/idle users/members
+ *
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ */
+
+const getIdleUsers = async (req, res) => {
+  try {
+    const allUsers = await userQuery.fetchUsers(req.query);
+    const onlyMembers = await members.fetchUsersWithRole(ROLES.MEMBER);
+    const taskParticipants = await tasks.fetchActiveTaskMembers();
```

Comment:
Can this be named `activeMembers` instead of `taskParticipants`?

Reply:
Yes, it will be more precise.